app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+  display: none;
+}
+
+.f-pnav__user-item-side-toggle {
+  cursor: pointer;
+  padding: 10px 10px 10px 5px;
+  margin-right: -10px;
+
+  .f-pnav__user-icon-bar {
+    background-color: $color-gray-600;
+    width: 20px;
+    display: block;
+    height: 2px;
+    border-radius: 1px;
+    @include transition_fast(background-color, margin);
+  }
+
+  .f-pnav__user-icon-bar + .f-pnav__user-icon-bar {
+    margin-top: 4px;
+  }
+
+  &:hover {
+    .f-pnav__user-icon-bar {
+      background-color: $color_forever_blue;
+    }
+
+    .f-pnav__user-icon-bar + .f-pnav__user-icon-bar {
+      margin-top: 6px;
+    }
+  }
+}
+
+.f-pnav__side {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  overflow: auto;
+  top: 0;
+  right: -($f-pnav__side--width);
+  bottom: 0;
+  z-index: $f-pnav__side--zindex;
+  width: $f-pnav__side--width;
+  margin: 0;
+  padding: 0 20px;
+  background-color: $color-white;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, 0.1);
+  direction: rtl;
+  text-align: left;
+  @include transition_normal(right);
+
+  &.is-onscreen {
+    right: 0;
+    width: 100%;
+  }
+}
+
+.f-pnav__side-section {
+  padding: 8px 0;
+  margin: 0;
+  list-style: none;
+  border-bottom: 2px solid $color-gray-200;
+  direction: ltr;
+
+  &:last-child {
+    border: none;
+  }
+
+  .f-pnav__side-signout-link,
+  .f-pnav__side-signout-link:hover {
+    color: $color-forever_red;
+  }
+
+  .f-pnav__side-link-base {
+    text-transform: uppercase;
+    display: block;
+    font-family: $font-face-semibold;
+    font-size: $font-size-200;
+    padding: 8px 0;
+  }
+
+}
+
+.f-pnav__side-section-close {
+  position: relative;
+  width: 22px;
+  padding-top: 20px;
+  align-self: flex-start;
+  cursor: pointer;
+  border: none;
+  outline: none;
+
+  &::before,
+  &::after {
+    position: absolute;
+    right: 0;
+    width: 22px;
+    height: 2px;
+    content: ' ';
+    border-radius: 1px;
+    background-color: $color-gray-600;
+    @include transition_fast(background-color);
+  }
+
+  &::before { transform: rotate(45deg); }
+  &::after { transform: rotate(-45deg); }
+
+  &:hover {
+    &::before,
+    &::after { background-color: $color-forever_blue; }
+  }
+}
+
+.f-pnav__side-section-info {
+  padding-bottom: 15px;
+
+  /* avatar/badge and storage meter */
+  .f-pnav__side-info-avameter {
+    display: flex;
+    align-items: center;
+  }
+
+  .f-pnav__side-info-avatar {
+    position: relative;
+    flex: 0 1 auto;
+
+    /* necessary to dig into avatar and add custom size*/
+    .avatar {
+      width: 80px;
+      height: 80px;
+    }
+
+    img {
+      position: absolute;
+      height: 50px;
+      width: 50px;
+      top: 48px;
+      left: 43px;
+    }
+  }
+
+  .f-pnav__side-info-storage-meter {
+    flex: 1 0 auto;
+    padding-left: 15px;
+
+    /*store only, mars has a component*/
+    .f-pnav__side-info-storage-meter-bg {
+      height: 10px;
+      border-radius: 50px;
+      background-color: $color-gray-300;
```

Comment:
if the background color is not being changed or part of a pseudo element it can always be set in the markup.